rbx_types/src/unique_id.rs

```diff
@@ -46,6 +46,18 @@ pub struct UniqueId {
 static INDEX: AtomicU32 = AtomicU32::new(0);
 
 impl UniqueId {
+    /// Returns a non-unique `UniqueId` that has every field set to `0`.
+    /// This value may appear multiple times in a Roblox file safely.
+    pub fn non_unique() -> Self {
+        Self {
+            index: 0,
+            time: 0,
+            random: 0,
+        }
+    }
```

Comment:
Can we just call this `nil`? It'd [mirror existing terminology for UUIDs](https://datatracker.ietf.org/doc/html/rfc4122.html#section-4.1.7). UniqueId is just a Roblox-brand UUID, and I think it's just more descriptive

Reply:
Sure, I wasn't aware that there was existing terminology for this and didn't want to use `null` because of the implications it has.